Modules/xASL_module_ASL.m

```diff
@@ -415,6 +415,8 @@
 		x.P.Path_PWI4D_used = x.P.Path_PWI4D;
 		x.P.Pop_Path_PWI4D_used = x.P.Pop_Path_PWI4D;
 	end
+	% Note: The paths Path_PWI4D_used can differ from a simple subtraction as it can point out to version that has some outlying volumes excluded.
+	% That's why we save this actual path used so that the correct version can be reused for xASL_wrp_VisualQC
```

Comment:
Change this note to:

% That's why we save and use x.P.(Pop_)Path_PWI4D_used in memory, such that the correct NIfTI will be used later in the pipeline (e.g., in xASL_wrp_VisualQC)

Reply:
Agree. Replaced